### ISSUE TITLE
Players list and Pause use more accessible elements for their text rather than headers

### ISSUE THREAD

**Issue by anathemamask** [foundryvtt/foundryvtt#5502]
Some HTML elements (such as the players menu and the game paused icon) use header `<h2> <h3>` tags to style those elements- per WCAG 2.1 these should be shifted to using alternate tags and header tags should be used only for actual headers to denote section breaks for screen readers.
